src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -34,11 +48,11 @@ public function __construct(ObjectManager $om, $doFlush = true)
     public function persist(array $objects)
     {
         foreach ($objects as $object) {
-            $this->om->persist($object);
+            $this->manager->persist($object);
         }
 
-        if ($this->flush) {
-            $this->om->flush();
+        if (true === $this->flush) {
```

Comment:
I think strict checking for true here is harmful indeed as it might break things that passed in 1 or whatever truthy/falsey value.